librepo/downloader.c

```diff
@@ -1334,7 +1334,7 @@ check_zck(LrTarget *target, GError **err)
         if(!cks_good) { // Error while validating checksums
             g_set_error(err, LR_DOWNLOADER_ERROR, LRE_ZCK,
                         "%s: Error validating zchunk file: %s", __func__,
-                        target->target->path);
+                        zck_get_error(zck));
```

Comment:
We already log the filename further up the stack, so the actual error message is more important here.